.codeclimate.yml

```diff
@@ -1,5 +1,11 @@
 version: "2"
 checks:
+  argument-count:
+    config:
+      threshold: 10
```

Comment:
After 5, I tend to think an object hash is in order. Totally don't want to debate this too heavily, though 😄.

Reply:
Yeah, I was just tired of Code Climate being so picky, but tend to agree 👍